docs/pages/get-started/installation.mdx

```diff
@@ -1,61 +1,56 @@
 ---
 title: Installation
-description: Learn how to get started creating a new Expo project quickly and easily using Expo CLI and Expo Go.
+description: Learn how to get started creating a new universal Expo project.
 ---
 
 import { Terminal } from '~/ui/components/Snippet';
 import { BoxLink } from '~/ui/components/BoxLink';
 import { ExpoGoLogo } from '@expo/styleguide';
 import { CALLOUT } from '~/ui/components/Text';
 
-To develop applications with Expo, you will want to start with two tools:
+### Requirements
 
-- [Expo CLI](#expo-cli) a command-line tool to serve your project
-- [Expo Go](/get-started/expo-go) a mobile client app to open the project on Android and iOS platforms.
+To use Expo, you need to have the following tools installed on your machine:
 
-Linux, macOS and Windows are all supported as your development machine. You can use any web browser to run the project on the web.
+- [**Node.js** LTS release](https://nodejs.org/en/) - Only Node.js LTS releases (even-numbered) are recommended.
+  <CALLOUT theme="secondary">
+    As Node.js [officially states](https://nodejs.org/en/about/releases/), "Production applications
+    should only use Active LTS or Maintenance LTS releases". You can install Node.js using a version
+    management tool (such as `nvm` or `volta` or any other of your choice) to switch between
+    different Node.js versions.
+  </CALLOUT>
+- [Git](https://git-scm.com) for source control.
+- [Watchman](https://facebook.github.io/watchman/docs/install#buildinstall) (for Linux or macOS users).
```

Comment:
Is watchman still always necessary? Seems fine when I don't install, get a lot less warnings, too, and it's a long install. The RN guide recommends it for better performance, but doesn't say it's required, FWIW. https://reactnative.dev/docs/environment-setup?guide=native

Reply:
we should recommend it for now. Afaik Metro is tested against the assumption that it's installed.